packages/ui/src/components/PreviewItem/PreviewItem.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Meta, StoryFn } from "@storybook/react";
+
+import PreviewItem from "./PreviewItem";
+import Chip from "../Chip/Chip";
+import { allModes } from "@gc-digital-talent/storybook-helpers";
+
+export const previewDetails = [
+  <Chip color="primary">Approved</Chip>,
+  "IT-01",
+  "Yonathan Kidanemariam",
+  <span>
+    Apply on or before:{" "}
+    <span data-h2-color="base(error)">April 30th, 2024</span>
+  </span>,
+];
+
+export default {
+  component: PreviewItem,
+  args: {
+    title: "Preview Item",
+    details: previewDetails,
+    buttonName: "View preview item",
+  },
+  parameters: {
+    layout: "fullscreen",
+    chromatic: {
```

Comment:
I'd recommend adding a mobile snapshot.

Reply:
:+1: Nice call, I think I got this in [eade550](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11192/commits/eade550b29ae61609adeb24609f4f1a1eca26baf)